Project.toml

```
@@ -1,12 +1,13 @@
 name = "TimeZones"
 uuid = "f269a46b-ccf7-5d73-abea-4c690281aa53"
 authors = ["Curtis Vogt <curtis.vogt@gmail.com>"]
-version = "1.2.0"
+version = "1.3.0"
```

You can leave this out of this PR. I'll bump the version number right before I make the release
```suggestion
version = "1.2.0"
```

---

Ok, in some repos they like if PR also bumps the version.